lib/vnet/vnet_test.go

```diff
@@ -480,13 +462,9 @@ func TestDialFakeApp(t *testing.T) {
 				"echo.myzone.example.com",
 				"echo.nested.myzone.example.com",
 				"not.in.a.custom.zone",
-				"in.an.invalid.zone",
 			},
 			customDNSZones: []string{
 				"myzone.example.com",
-				// This zone will not have a valid TXT record because it is not included in the
-				// customDNSZones passed to newTestPack.
-				"an.invalid.zone",
```

Comment:
After removing the TXT record requirement, this custom domain becomes yet another custom DNS zone in use. It's not that much different in behavior to `myzone.example.com` above, so I just removed it.